lib/reek/smells/uncommunicative_variable_name.rb

```diff
@@ -24,13 +26,17 @@ class UncommunicativeVariableName < SmellDetector
       # The name of the config field that lists the regexps of
       # smelly names to be reported.
       REJECT_KEY = 'reject'.freeze
-      DEFAULT_REJECT_SET = [/^.$/, /[0-9]$/, /[A-Z]/].freeze
+      DEFAULT_REJECT_SET = [
+        /^[a-z]$/, # single-character names
```

Comment:
Since Ruby allows loads of unicode characters in variable names, I think this is too narrow. This detector will now happily accept the variable name `é`.